src/account.ts

```diff
@@ -153,7 +158,7 @@ export class Account {
      * @param receiverId NEAR account receiving the transaction
      * @param actions list of actions to perform as part of the transaction
      */
-    protected async signAndSendTransaction(receiverId: string, actions: Action[]): Promise<FinalExecutionOutcome> {
+    async signAndSendTransaction(receiverId: string, actions: Action[]): Promise<FinalExecutionOutcome> {
```

Comment:
this is potentially a breaking change  as anyone extending the `Account` class with a custom implementation of `signAndSendTransaction` will have to remove the `protected` accessor.

This is fine, and something that is often requested anyways. But we need to make not of this before we publish this.

Reply:
@mehtaphysical I don't think anybody besides wallet is doing it for now, do you know of any use cases? Also kinda trivial change for them anyway (and doesn't matter for JS at all, so it only affects peeps who rebuild from TS).

Reply:
Agree. It seems pretty unlikely that someone is re-implementing this. Also, very true it only matters to TS projects.

@raymondfeng is the only person I know of who's been messing around with this. And he initially requested a change like  this anyways.

Reply:
Also, I think this is a good change. Just need to make sure that we make note of it in the release notes.